packages/inventory-insights/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@redhat-cloud-services/frontend-components-inventory-insights",
-    "version": "0.4.2",
+    "version": "1.0.1",
```

Comment:
No need to update the version, bots should take care of it.

Reply:
i knooooooowwwww but hate to see this field outta sync with publish field.... 
maybe we'll just can it next time 🤔